concepticondata/conceptlists.tsv

```diff
@@ -1,4 +1,5 @@
 ID	AUTHOR	YEAR	LIST_SUFFIX	ITEMS	TAGS	SOURCE_LANGUAGE	TARGET_LANGUAGE	URL	REFS	PDF	NOTE	PAGES	ALIAS
+Brid-2022-324	Brid, Nicolás and Messineo, Cristina and List, Johann-Mattis	2022		324	basic	English	Chaco Languages	https://zenodo.org/record/6656484#.Y5NIpn3MK3A	Brid2022		The concept list is taken from a dataset on Chaco Languages.	
```

Comment:
The URL should be the DOI on Zenodo, not a zenodo link, please check for the DOI online, they usually share it there.